i18n/zh/docusaurus-plugin-content-docs/current/architecture/cloud/device_controller.md

```diff
@@ -2,70 +2,61 @@
-|Device Node Membership Updated | The device controller sends a membership update event to the edge node.|
-|Device  Twin Desired State Updated | The device controller sends a twin update event to the edge node.|
-|Device Deleted                 | The controller sends the device twin delete event to delete all device twins associated with the device. It also deletes config maps associated with the device and this delete event is synced to the edge. The mapper application effectively stops operating on the device.|
+| 更新类型                           | 操作                                         |
+|-------------------------------|----------------------------------------------|
+| **新设备模型创建**                 | NA                                          |
+| **新设备创建**                    | Device Controller创建一个新的 config map 来存储设备模型中定义的设备属性和访问者信息，并将其存储在 etcd 中。Edge Controller中现有的 config map 同步机制用于将 config map 同步到边缘节点。运行在容器中的 mapper 应用程序可以获取更新的 config map，并使用属性和访问者元数据访问设备。Device Controller还将Device Twin元数据更新报告给Edge node。|
+| **设备节点成员更新**               | Device Controller向Edge node发送成员更新事件。|
+| **Device Twin期望状态更新**           | Device Controller向Edge node发送孪生更新事件。|
+| **设备删除**                       | Device Controller发送Device Twin删除事件以删除与设备关联的所有设备孪生。它还删除与设备关联的 config map，并将此删除事件同步到边缘。mapper 应用程序将停止操作该设备。|
 
 ![Device Downstream Controller](/img/device-crd/device-downstream-controller.png)
 
-The idea behind using config map to store device properties and visitors is that these metadata are only required by the mapper applications running on the edge node in order to connect to the device and collect data.
-Mappers if run as containers can load these properties as config maps. Any additions, deletions, or updates to properties, visitors and etc in the cloud are watched upon by the downstream controller and config maps are updated in etcd.
-If the mapper wants to discover what properties a device supports, it can get the model information from the device instance.
-Also, it can get the protocol information to connect to the device from the device instance. Once it has access to the device model,
-it can get the properties supported by the device. In order to access the property, the mapper needs to get the corresponding visitor information.
-This can be retrieved from the propertyVisitors list. Finally, using visitorConfig, the mapper can read/write the data associated with the property.
+使用config map存储设备属性和访问者的想法是，这些元数据仅由运行在Edge node上的mapper应用程序所需，以便连接到设备并收集数据。如果mapper作为容器运行，可以将这些属性作为config map 加载。下行Controller监视云端对属性、访问者等的添加、删除或更新，并在etcd中更新config map。如果mapper想要发现设备支持哪些属性，可以从设备实例中获取模型信息。此外，它还可以从设备实例中获取连接设备的协议信息。一旦它可以访问设备模型，就可以获取设备支持的属性。为了访问属性，mapper需要获取相应的访问者信息。这可以从propertyVisitors列表中检索。最后，使用 visitorConfig，mapper可以读取/写入与属性相关的数据。
```

Comment:
Using `ConfigMap` to replace `config map`

Reply:
Modified.